### ISSUE TITLE
Tests in error and Unable to connect to Elasticsearch service

### ISSUE THREAD

**Issue by jshankarrepo** [fabric8io/gofabric8#176]
Any sample project created is not running in Kubernetes pods 
Failed to pull image "10.0.0.123:80/example/fabric8-demo:1.0.3":
this may be because there are no credentials on this request. details: (Error response from daemon: Get https://10.0.0.123:80/v1/_ping: tls: oversized record received with length 20527)"

Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 315.756 sec <<< FAILURE! - in org.example.KubernetesIntegrationKT
org.example.KubernetesIntegrationKT  Time elapsed: 315.753 sec  <<< ERROR!
java.lang.RuntimeException: java.lang.IllegalStateException: Failed to apply kubernetes configuration.
33mWaiting for container:spring-boot. Reason:ErrImagePull[m
[33mWaiting for container:spring-boot. Reason:ErrImagePull[m
[33mWaiting for container:spring-boot. Reason:ErrImagePull[m

**Comment by rawlingsj** [comment]
The 'Unable to connect to elasticsearch' message is just for information as typically people run the centralised logging app which means we get automated pipeline events sent to elasticsearch for nice visualisations.  It might be best if we don't log that message to the console as it can be confusing whether or not things are working.

The integration test failure I think was fixed yesterday but I'm just recreating my local minikube environment to test.  I'll come back and confirm shortly.

**Comment by jshankarrepo** [comment]
@rawlingsj Thanks you very much! I am waiting for the confirmation.

**Comment by rawlingsj** [comment]
@jshankarrepo it looks like we had an issue with our release infra yesterday so the latest fixes we're released.  I've just kicked off another job so will report back after that.

**Comment by rawlingsj** [comment]
@jshankarrepo I just tried with the latest release on minikube, creating a new microservices app and it worked.

![screenshot 2016-09-15 12 17 59](https://cloud.githubusercontent.com/assets/1457180/18548046/96987a8c-7b3e-11e6-9515-ee6eb7ffe865.png)

Until we have a clean up or update command I ran,

```
kubectl delete deployments --all
kubectl delete namespace default-testing # this removed the failing pod from the previous release 
gofabric8 deploy
```

And created a new project.  Fingers crossed this will get you a green pipeline.
